tools/system_libs.py

```diff
@@ -1041,8 +1040,7 @@ def __init__(self, **kwargs):
   def get_cflags(self):
     cflags = ['-pthread',
               '-D_DEBUG' if self.debug else '-Oz',
-              '-DSTACK_OVERFLOW_CHECK=' + ('2' if self.stack_check else '0'),
-              '-DWASM_WORKER_NO_TLS=' + ('0' if self.tls else '1')]
+              '-DSTACK_OVERFLOW_CHECK=' + ('2' if self.stack_check else '0')]
```

Comment:
Can you also remove the `STACK_OVERFLOW_CHECK` variant?   As far as I can tell there is no usage of the `STACK_OVERFLOW_CHECK` macro in the C source code.. unless I'm missing something?

Reply:
I can look into that as a separate PR.